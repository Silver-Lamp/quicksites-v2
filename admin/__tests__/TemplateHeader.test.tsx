import { render } from '@testing-library/react';
import TemplateHeader from '../components/templates/TemplateHeader';

it('matches snapshot', () => {
  const { asFragment } = render(
    <TemplateHeader
      name="snapshot-template"
      updatedAt="2025-05-24T20:00:00Z"
      duplicateUrl="/admin/templates-new?copy=snapshot-template"
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
