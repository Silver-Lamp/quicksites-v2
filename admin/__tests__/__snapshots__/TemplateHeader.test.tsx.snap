// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex justify-between items-center mb-4"
  >
    <div>
      <h3
        class="text-lg font-semibold"
      >
        snapshot-template
      </h3>
      <p
        class="text-xs text-muted-foreground"
      >
        Last updated: 5/24/2025, 1:00:00 PM
      </p>
    </div>
    <a
      href="/admin/templates-new?copy=snapshot-template"
    >
      <button
        class="text-sm text-blue-500 hover:underline"
      >
        Duplicate Template
      </button>
    </a>
  </div>
</DocumentFragment>
`;
